Makefile

```diff
@@ -33,7 +33,8 @@ test-trigger: ## Runs unit tests for the pipeline trigger code
 	poetry run python -m pytest tests/trigger
 
 compile-pipeline: ## Compile the pipeline to training.json or prediction.json. Must specify pipeline=<training|prediction>
-	@cd pipelines/src && \
+	@. ./env.sh && \
```

Comment:
this should be covered by line 15 `-include env.sh`

Reply:
It doesn't expand bash expressions unfortunately :( I've had to replace by adding conditional assignments

```
RESOURCE_SUFFIX ?= $(shell . env.sh && echo $$RESOURCE_SUFFIX)
PIPELINE_FILES_GCS_PATH ?= $(shell . env.sh && echo $$PIPELINE_FILES_GCS_PATH)
```

At the top of the Makefile

Reply:
if this is just getting awkward and annoying I suggest we remove the git username functionality and just have the user specify a suffix

Reply:
Will do that, didn't realise but after fixing e2e tests, the local functionality stopped working 💀